pkg/ccl/changefeedccl/changefeed_processors.go

```diff
@@ -758,6 +776,15 @@ func (ca *changeAggregator) Next() (rowenc.EncDatumRow, *execinfrapb.ProducerMet
 			break
 		}
 
+		select {
+		case <-ca.aggTimer.C:
+			ca.aggTimer.Read = true
+			ca.aggTimer.Reset(1 * time.Second)
```

Comment:
could be worth following convention to reset this timer every 15 seconds, so it intercepts with next() less frequently